docs/designs/reference_link.md

```diff
@@ -0,0 +1,194 @@
+[\\]: # (TODO: correct the links once the corresponding docs have been added)
```

Comment:
We can also make it clear where the path should be URL encoded later in this document.

Reply:
sure, one more thing need to be added is the frgement and query string support

---
In reply to: [187233768](https://github.com/dotnet/docfx/pull/2752#discussion_r187233768) [](ancestors = 187233768)